src/queries/customers.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @name customers
+ * @method
+ * @memberof Accounts/NoMeteorQueries
+ * @summary Returns accounts optionally filtered by group IDs
+ * @param {Object} context - an object containing the per-request state
+ * @returns {Promise} Mongo cursor
+ */
+export default async function customers(context) {
+  const { collections } = context;
+  const { Accounts } = collections;
+
+  await context.validatePermissions("reaction:legacy:accounts", "read");
+
+  return Accounts.find({
+    groups: []
```

Comment:
Might be safer to do `groups: { $in: [null, []] }` to handle `null` or missing `groups` field?